sdk/eventhub/Microsoft.Azure.EventHubs/tests/Processor/ProcessorTestBase.cs

```diff
@@ -18,398 +18,446 @@ namespace Microsoft.Azure.EventHubs.Tests.Processor
 
     public class ProcessorTestBase
     {
-        protected string LeaseContainerName;
-        protected string[] PartitionIds;
-
-        public ProcessorTestBase()
+        //Unused after Track two test infrastructure ported
+        //protected string LeaseContainerName;
+        //protected string[] PartitionIds;
+
+        //Unused after Track two test infrastructure ported
+        //public ProcessorTestBase()
+        //{
+        //    // Use entity name as lease container name.
+        //    // Convert to lowercase in case there is capital letter in the entity path.
+        //    // Uppercase is invalid for Azure Storage container names.
+        //    this.LeaseContainerName = new EventHubsConnectionStringBuilder(TestUtility.EventHubsConnectionString).EntityPath.ToLower();
+
+        //    // Discover partition ids.
+        //    TestUtility.Log("Discovering partitions on eventhub");
+        //    var ehClient = EventHubClient.CreateFromConnectionString(TestUtility.EventHubsConnectionString);
+        //    var eventHubInfo = ehClient.GetRuntimeInformationAsync().WaitAndUnwrapException();
+        //    this.PartitionIds = eventHubInfo.PartitionIds;
+        //    TestUtility.Log($"EventHub has {PartitionIds.Length} partitions");
+        //}
+
+        //update method to use EventHub managed by EventHubScope
+        public String[] ProcessorPartitionIds(string connectionString)
         {
-            // Use entity name as lease container name.
-            // Convert to lowercase in case there is capital letter in the entity path.
-            // Uppercase is invalid for Azure Storage container names.
-            this.LeaseContainerName = new EventHubsConnectionStringBuilder(TestUtility.EventHubsConnectionString).EntityPath.ToLower();
-
-            // Discover partition ids.
-            TestUtility.Log("Discovering partitions on eventhub");
-            var ehClient = EventHubClient.CreateFromConnectionString(TestUtility.EventHubsConnectionString);
+            var ehClient = EventHubClient.CreateFromConnectionString(connectionString);
             var eventHubInfo = ehClient.GetRuntimeInformationAsync().WaitAndUnwrapException();
-            this.PartitionIds = eventHubInfo.PartitionIds;
-            TestUtility.Log($"EventHub has {PartitionIds.Length} partitions");
+           String[] PartitionIds = eventHubInfo.PartitionIds;
+            return PartitionIds;
         }
 
+        //Update Live test to use EventHub managed by EventHubScope
         /// <summary>
         /// Validating cases where entity path is provided through eventHubPath and EH connection string parameters
         /// on the EPH constructor.
         /// </summary>
         [Fact]
         [LiveTest]
         [DisplayTestMethodName]
-        public void ProcessorHostEntityPathSetting()
+        public async Task ProcessorHostEntityPathSetting()
```

Comment:
I may be missing something, but does this need to be a `[LiveTest]`?  I don't see it actually using the scope at all, just testing local artifacts.   If not, let's please remove the attribute and scope so this runs as a standard unit test.

Reply:
Yes, this not really use Azure service. Undo the change and remove [LiveTest] attribute.